Source/Core/DolphinQt2/WiiUpdate.cpp

```diff
@@ -74,46 +119,25 @@ void PerformOnlineUpdate(const std::string& region, QWidget* parent)
               true);
           return !was_cancelled.IsSet();
         },
-        region);
+        std::forward<Args>(args)...);
     Core::QueueHostJob([&dialog] { dialog.close(); }, true);
     return res;
   });
 
   dialog.exec();
+  Core::HostDispatchJobs();
```

Comment:
I think this was added back when I first noticed the cancel button logic was broken (which caused Qt to return earlier than expected from dialog.exec()). I have removed this.